src/std-data/model.cmake

```diff
@@ -50,4 +65,13 @@ macro(setup)
         register_definitions(USE_ONEDPL)
         register_link_library(oneDPL)
     endif ()
+    if (CLANG_OFFLOAD)
+        set(CLANG_FLAGS --hipstdpar --hipstdpar-path=${CLANG_STDPAR_PATH} --offload-arch=${CLANG_OFFLOAD})
```

Comment:
Done in commit [9f67c5f0c](https://github.com/gsitaram/BabelStream/pull/1/commits/9f67c5f0ca38cbeca701fa82d4c2ab6acef31431) we hope this will be used with ROCm 6.1 onwards.